M/ModularForms/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+using BinaryBuilder, Pkg
+import Pkg.Types: VersionSpec
+
+name = "ModularForms"
+version = v"0.1.0"
+
+sources = [GitSource("https://gitlab.com/mraum/ModularForms_jll_src.git",
+                     "4d81d41e625fde3e36dc86df5061313a8447cacd")]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/ModularForms_jll_src
+if [[ ${target} == *musl* ]]; then
+   export CFLAGS=-D_GNU_SOURCE
+fi
+./configure --prefix=$prefix --disable-static --enable-shared --with-gmp=$prefix --with-mpfr=$prefix --with-flint=$prefix --with-arb=$prefix CC=gcc ${extraflags}
+make -j${nproc} verbose
```

Comment:
Compilation for Windows adds a bunch of references to `/workspace/destdir/lib`, all of which are wrong since the libraries are in `/workspace/destdir/bin`

Reply:
We do not have the possibility to test on anything but Linux. I don't understand the Windows ecosystem; So would it be acceptable to just leave Linux in the platforms?

Reply:
We prefer to build for as many platforms as possible, but the final decision of which platforms to support is yours.  But I also already  told you how to fix the failure for the Windows build: the libraries are in `${prefix}/bin`, not `${prefix}/lib`.  Or just always use `${libdir}` from our environment which always points to the right directory.